index.html

```diff
@@ -260,11 +260,12 @@ <h1>Contact me</h1>
                 I'd love to help with it </p>
             </div>
             <div class="contact-form">
-            <form action="https://formspree.io/html/" method="post">
+            <form action="https://formspree.io/f/xnqwordb" method="post">
                 <input name="first_name" id="text" type="text" maxlength="30" placeholder="Afolabi" required>
                 <input name="Email" id="email" type="email" placeholder="example@gmail.com" required>
-                <input type="text" id="message" maxlength="500" placeholder="write your message here" required>
-                <button type="submit" class="btn">Get It Touch</button>
+                <textarea id="message" name="message" maxlength="500" required>write your message here
+                </textarea>
+                <button type="submit" class="btn">Get in Touch</button>
             </form>
             </div>
             <div class="contact-shape">
```

Comment:
- [ ] Please wrap this section inside `<footer>`, This will make sure the code is semantic and follow best practices.